force_bdss/core/verifier.py

```diff
@@ -2,195 +2,43 @@
 
 from itertools import groupby
 
-from traits.api import HasStrictTraits, Unicode, Any
+from traits.api import Any, Enum, HasStrictTraits, Unicode
 
 logger = logging.getLogger(__name__)
 
 
 class VerifierError(HasStrictTraits):
-    subject = Any()
-    #: An error message relevant to the local modelview
-    local_error = Unicode()
-    #: An error message relevant to the overall workflow
-    global_error = Unicode()
 
-    def __init__(self, subject, global_error='', local_error=''):
-        if local_error == '':
-            local_error = global_error
+    #: How severe the error is.
+    severity = Enum('error', 'warning', 'information')
 
-        super(VerifierError, self).__init__(
-            subject=subject,
-            global_error=global_error,
-            local_error=local_error
-        )
+    #: The object that has an error.
+    subject = Any
+
+    #: An optional trait that holds the error value.
```

Comment:
Could we extend the docstring here to explain that this refers to a trait name within `subject`. At first I wasn't sure what a error value was until seeing it in use